src/subscription/data/details/selectors.js

```diff
@@ -0,0 +1,76 @@
+import { getConfig, getQueryParameters } from '@edx/frontend-platform';
+import { createSelector } from 'reselect';
+import Cookies from 'universal-cookie';
+
+import { storeName } from '../constants';
+
+export const localizedCurrencySelector = () => {
+  const cookie = new Cookies().get(getConfig().CURRENCY_COOKIE_NAME);
+  let currencyCode;
+  let conversionRate;
+
+  if (cookie && typeof cookie.code === 'string' && typeof cookie.rate === 'number') {
+    currencyCode = cookie.code;
+    conversionRate = cookie.rate;
+  }
+
+  const showAsLocalizedCurrency = typeof currencyCode === 'string' ? currencyCode !== 'USD' : false;
+
+  return {
+    currencyCode,
+    conversionRate,
+    showAsLocalizedCurrency,
+  };
+};
+
+export const subscriptionDetailsSelector = state => state[storeName].details;
+
+export const detailsSelector = createSelector(
+  subscriptionDetailsSelector,
+  localizedCurrencySelector,
+  (details, currency) => ({
+    ...details,
+    isCurrencyConverted: currency.showAsLocalizedCurrency,
+  }),
+);
+
+export const currencyDisclaimerSelector = state => ({
+  actualAmount: state[storeName].details.orderTotal,
+});
+
+export const updateQuantityFormSelector = createSelector(
```

Comment:
Was this selector copied from the regular flow? We won't have a quantity form for subscriptions

Reply:
removed.